src/js/juju-cards-v1.0.0.js

```diff
@@ -92,7 +93,7 @@ let jujuCards = () => {
         `</main>` +
         `<footer class="bundle-card__footer">` +
           `<a href="http://jujucharms.com"><img src="https://jujucharms.com/static/img/logos/logo.svg" alt="" class="bundle-card__footer-logo" /></a>` +
-          `<p class="bundle-card__footer-note">© 2015 <a href="http://www.canonical.com">Canonical Ltd</a>.</p>` +
+          `<p class="bundle-card__footer-note">&copy; ${currentYear} <a href="http://www.canonical.com">Canonical Ltd</a>.</p>` +
```

Comment:
Is this dynamic year at the request of someone in Legal? Or are you just doing it 'cos you can? 'Cos actually updating the year [*weakens* any protections](http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated) that the copyright notice would give us. Although I don't have much respect for copyright symbols anyway, if we don't want to specify a year, I'd just remove it. Copyright is [pretty much implied anyway](https://www.firstwriter.com/copyright/symbol.shtml).